app/assets/javascripts/search.js

```diff
@@ -75,13 +75,19 @@ $(document).ready(function () {
   function render() {
     
     state.feedback.forEach(function (value) {
-      var tr = state.table.append('<tr></tr>');
```

Comment:
This will give an empty element (with no content) and columns <td> elements were coming outside of <tr> which should not be the case.